assets/js/script.js

```diff
@@ -143,7 +143,10 @@ function checkResult() {
 
 function isWinner(a,b){
     if (a == 3 || b == 3) {
+        
         document.getElementById("result").innerHTML = a == 3 ? "Player 1 Win" : "Player 2 Win";
+        if(document.getElementById("result").innerHTML=="Player 1 Win")  alert("Player 1 Win Hurray !")
+        else alert("Player 2 win Hurray !");
```

Comment:
As soon as any player wins it first give alert message and then update the box for X or O. after that innerhtml of result update as follows `document.getElementById("result").innerHTML = a == 3 ? "Player 1 Win" : "Player 2 Win";`

As soon as click box located at 3rd row 1st column it gives winning alert
![image](https://github.com/malivinayak/Tic-Tac-Toe/assets/66154908/1a20a20d-5844-454f-97b2-fac83428d9df)

After Clicking OK, it update the X or O sign in box
![image](https://github.com/malivinayak/Tic-Tac-Toe/assets/66154908/7aaec500-e106-4c30-828f-c97a36032ad3)

Reply:
This issue is not resolved yet
Checkout the video attached to latest comment